.dependabot/config.yml

```diff
@@ -0,0 +1,29 @@
+version: 1
+update_configs:
+
+  - package_manager: docker
+    directory: /
+    update_schedule: daily
+    commit_message:
+      prefix: chore
+      include_scope: true
+    default_labels:
+      - ':recycle: Chore'
+
+  - package_manager: javascript
+    directory: /
+    update_schedule: live
```

Comment:
Might be too much.

Reply:
Like, everything every day all the time?

Reply:
Yeah, rather than getting a batch once a day, bigger batch once a week or large batch once a month. (Not sure how Dependabot handles update depths, if it only updates top-level dependencies then not much of a problem, but if there's individual PRs for _everything_ in `package-lock.json`...)

Reply:
A common side-effect is that build time is spent on testing the updates, so needs batching at the moment to avoid Travis CI being a bottleneck; especially if there are many PRs open as they are all rebased continuously to resolve conflicts. If moving to Github Actions the bottleneck may become code review time.